pulp.spec

```diff
@@ -458,13 +458,13 @@ Pulp provides replication, access, and accounting for software repositories.
 # If we are upgrading
 if [ $1 -gt 1 ] ; then
     %if %{pulp_systemd} == 1
-        /bin/systemctl stop pulp_workers > /dev/null 2>&1
-        /bin/systemctl stop pulp_celerybeat > /dev/null 2>&1
-        /bin/systemctl stop pulp_resource_manager > /dev/null 2>&1
+        /bin/systemctl stop pulp_workers > /dev/null 2>&1 || :
```

Comment:
Since this is only seen on RHEL6 (presumably because systemd is more robust than our init scripts and just does the right thing) do we need this on the systemctl versions?